File: cpp/g10.parent/Student.cpp
#include "Student.h"

namespace PersonSpace {

 // Constructors
 Student::Student(const char* n)           : Person(n) { /* nothing */ }
 Student::Student(const char* n, double g) : Person(n) { setGPA(g); }

 // Methods
 void   Student::setGPA(double g) { gpa = g; } // mutator
 double Student::getGPA()         { return gpa; }   // accessor

}

File: cs/g10.parent/Student.cs
public class Student : Person
{
  private double gpa;

  /******* CONSTRUCTORS *******/
  public Student(string name)             { this.Name = name; }
  public Student(string name, double gpa) { this.Name = name; this.GPA = gpa; }

  /******* PROPERTIES *******/
  public double GPA {
      get { return gpa; }    // accessor - get data
      set { gpa = value; }   // mutator  - set data
  }
}
